contrib/test_jenkins.sh

```diff
@@ -754,6 +788,19 @@ test_malloc_hook() {
 	fi
 }
 
+test_jucx() {
+	echo "==== Running jucx test ===="
+	echo "1..2" > jucx_tests.tap
+	if module_load dev/jdk && module_load dev/mvn
+	then
+		cd ../bindings/java && mvn test
```

Comment:
need to keep in build dir, not go to source dir
and also restore to prev dir, to not break other tests
same for build_java_docs